etcdserver/api/v3rpc/lease.go

```diff
@@ -107,7 +110,13 @@ func (ls *LeaseServer) leaseKeepAlive(stream pb.Lease_LeaseKeepAliveServer) erro
 			return nil
 		}
 		if err != nil {
-			plog.Debugf("failed to receive lease keepalive request from gRPC stream (%q)", err.Error())
+			ev, _ := status.FromError(err)
```

Comment:
@spzala Let's wrap this as a separate function inside `etcdserver/api/v3rpc`.

```go
func isClientCtxErr(err error) error {
  ev, _ := status.FromError(err)
  ...
  return true // when code == codes.Canceled || code == codes.DeadlineExceeded
}
```

Reply:
OK, yup, makes sense specially that we are doing for all of the messages. Thanks!